core/client/fs/src/main/java/alluxio/client/file/AlluxioFileInStream.java

```diff
@@ -147,6 +147,10 @@ public int read() throws IOException {
         if (result != -1) {
           mPosition++;
         }
+        if (mBlockInStream.remaining() == 0) {
+          closeBlockInStream(mBlockInStream);
+          mBlockInStream = null;
```

Comment:
The setting of `mBlockInStream = null` is handled in `closeBlockInStream`.

Reply:
removed this line.